docs/what-is-datahub/datahub-concepts.md

```diff
@@ -0,0 +1,111 @@
+# DataHub Concepts
+
+Explore key concepts of DataHub to take full advantage of its capabilities in managing your data.
+
+## General Concepts
+
+### URN (Uniform Resource Name)
+URN (Uniform Resource Name) is the chosen scheme of URI to uniquely define any resource in DataHub. It has the following form.
+```
+urn:<Namespace>:<Entity Type>:<ID>
+```
+> * [What is URN?](/docs/what/urn.md)
+
+### Entity
+An entity is the primary node in the metadata graph. For example, an instance of a Dataset or a CorpUser is an Entity. 
+
+> * [How does DataHub model metadata?](/docs/modeling/metadata-model.md)
+
+### Aspect
+An aspect is a collection of attributes that describes a particular facet of an entity. 
+Aspects can be shared across entities, for example "Ownership" is an aspect that is re-used across all the Entities that have owners. 
+
+> * [What is a metadata aspect?](/docs/what/aspect.md)
+> * [How does DataHub model metadata?](/docs/modeling/metadata-model.md)
+
+### Policy
+Access policies in DataHub define who can do what to which resources. 
+
+> * [Policies Guide](/docs/authorization/policies.md)
+> * [Developer Guides: DataHubPolicy](/docs/generated/metamodel/entities/dataHubPolicy.md)
+> * [Feature Guides: About DataHub Access Policies](/docs/authorization/access-policies-guide.md)
+
+### Role
+DataHub provides the ability to use Roles to manage permissions.
+
+> * [About DataHub Roles](/docs/authorization/roles.md)
+> * [Developer Guides: DataHubRole](/docs/generated/metamodel/entities/dataHubRole.md)
+
+
+### Relationships 
+A relationship represents a named edge between 2 entities. They are declared via foreign key attributes within Aspects along with a custom annotation (@Relationship). 
+
+> * [What is a relationship?](/docs/what/relationship.md)
+> * [How does DataHub model metadata?](/docs/modeling/metadata-model.md)
+
+## Metadata Models
+
+### Data platform 
+Data Platforms are systems or tools that contain Datasets, Dashboards, Charts, and all other kinds of data assets modeled in the metadata graph.
+Examples of data platforms are redshift, hive, bigquery, looker, tableau etc.
```

Comment:
We should actually link to a full list of data platform "names" - this is really useful for multiple places throughout our code. Here is the default list of data platforms that DataHub ships with: https://github.com/acryldata/datahub-fork/blob/acryl-main/metadata-service/war/src/main/resources/boot/data_platforms.json

Maybe this can be a separate page called "Data Platforms", or can be a collapsible table inside this page